tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -724,16 +740,22 @@ async function waitForSubscriptionsTillReady(targetNamespace) {
   await waitForSubscription('order-created', targetNamespace);
 }
 
-async function checkInClusterEventDelivery(targetNamespace) {
-  await checkInClusterEventDeliveryHelper(targetNamespace, 'structured');
-  await checkInClusterEventDeliveryHelper(targetNamespace, 'binary');
-  await checkInClusterLegacyEvent(targetNamespace);
+async function checkInClusterEventDelivery(targetNamespace, testV1Alpha2CRD=false) {
+  await checkInClusterEventDeliveryHelper(targetNamespace, 'structured', testV1Alpha2CRD);
+  await checkInClusterEventDeliveryHelper(targetNamespace, 'binary', testV1Alpha2CRD);
+  await checkInClusterLegacyEvent(targetNamespace, testV1Alpha2CRD);
 }
 
 // send event using function query parameter send=true
-async function sendInClusterEventWithRetry(mockHost, eventId, encoding, retriesLeft = 10) {
+async function sendInClusterEventWithRetry(mockHost, eventId, encoding, eventType='', retriesLeft = 10) {
```

Comment:
Have you changed all places calling this function to respect its new signature?

Reply:
yes